build.gradle

```diff
@@ -27,6 +28,7 @@ repositories {
 
     jcenter()
     mavenCentral()
+    mavenLocal()
```

Comment:
Do you need maven local?

Reply:
That the trick to be able to test common changes locally in other projects.

Reply:
👍